File: mvn-advent-of-code/src/test/java/advent/twenty_seventeen/Day06MemoryReallocationTest.java
package advent.twenty_seventeen;

import advent.common.DailyProblem;
import org.junit.Test;

import static org.junit.Assert.*;

public class Day06MemoryReallocationTest {
    @Test
    public void checkReal() {
        DailyProblem<Integer, Integer> dp = new Day06MemoryReallocation("/twenty_seventeen/Day06-MemoryReallocation-input.txt");

        assertEquals(3156, dp.getPart1Answer().intValue());
        assertEquals(1610, dp.getPart2Answer().intValue());
    }

    @Test
    public void checkExample1() {
        DailyProblem<Integer, Integer> dp = new Day06MemoryReallocation("/twenty_seventeen/Day06-MemoryReallocation-example#1.txt");

        assertEquals(5, dp.getPart1Answer().intValue());
        assertEquals(4, dp.getPart2Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_seventeen/Day06MemoryReallocationTest.scala
package advent.twenty_seventeen

import org.scalatest.{FlatSpec, Matchers}

class Day06MemoryReallocationTest  extends FlatSpec with Matchers {
  "2017 Day 06 - Input" should "calculate answers" in {
    val dp = new Day06MemoryReallocation("/twenty_seventeen/Day06-MemoryReallocation-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day06MemoryReallocation("/twenty_seventeen/Day06-MemoryReallocation-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}
